src/view/shell/Drawer.tsx

```diff
@@ -64,6 +64,10 @@ export const DrawerContent = observer(function DrawerContentImpl() {
       const state = navigation.getState()
       store.shell.closeDrawer()
       if (isWeb) {
+        // hack because we have flat navigator for web and MyProfile does not exist on the web navigator -ansh
+        if (tab === 'MyProfile') {
+          navigation.navigate('Profile', {name: store.me.handle})
+        }
```

Comment:
Need an else here, yeah?

Reply:
oops yeah thanks for the catch

Reply:
Fixed! @pfrazee Pls approve so I can merge

Reply:
@pfrazee Done, can you approve so I can merge?